src/transactions/ContentController.js

```diff
@@ -320,6 +318,14 @@ class ContentController {
     this.updateEditorState(withNewBlock, newSelection);
     return this;
   }
+  splitBlock = () => {
+    this.updateEditorState(
+      Modifier.splitBlock(
+        this.currentContent,
+        this.selection
+      )
+    );
```

Comment:
it must return this

Reply:
to make it more uniform lets make it 
`const splittedContent = Modifier.splitBlock(
         this.currentContent,
         this.selection
       )
this.updateEditorState(splittedContent)
return this
`